### ISSUE TITLE
Soc Learning objectives don't get module title headers

### ISSUE THREAD

**Issue by oscryan** [openstax/tutor#631]
### Description
When imported into Tutor, chapter-level learning objectives are separated by module but not labeled as to which module they came from.

**To Reproduce**
 Go to https://tutor-content.openstax.org/book/134/page/94004

**Expected behavior**
Module titles should appear before each module's learning objectives.

**Screenshots**
From tutor
<img width="587" alt="Screen Shot 2019-10-02 at 10 23 30 AM" src="https://user-images.githubusercontent.com/1895432/66057781-e8cbb080-e4fe-11e9-90ae-40b9bfa0a976.png">

From CNX https://cnx.org/contents/AgQDEnLI:TrIRM88K
<img width="688" alt="Screen Shot 2019-10-02 at 10 23 41 AM" src="https://user-images.githubusercontent.com/1895432/66057836-fd0fad80-e4fe-11e9-9615-aacb99c98abc.png">

**Additional context**

### Acceptance Tests
Additional acceptance tests that are required by this bug fix.

**title**: template  
**categories**: Interactive Component, Navigation, TOC, Math, Book Content, Other
> GIVEN something  
> AND something else  
> WHEN something  
> AND something else  
> THEN something  
> AND something else  

### Checklist for Done
2-CODE
  - [ ] A pull request is opened and linked to this issue. The automated pull request checks pass. 
  - [ ] The change has been approved by other developers.
  - [ ] The pull request is merged into master, and the change branch is deleted.
  - [ ] If there are new acceptance tests, they are finalized in "Given / When / Then" format.
  - [ ] Regression test categories are identified on the issue. Categories should be referenced if any changes might affect them, even if the intended functionality is unchanged.
- [ ] Add milestone tag
  
4A-UX REVIEW
- [ ] If UI changes, UX reviews and verifies. 

5A-FUNCT VER
  - [ ] All the acceptance tests for this issue have passed.
 
5C-REGRESSION
  - [ ] The test plan in testrail passes

**Comment by carlycurran** [comment]
Ryan, are you looking at this in Tutor-content? And is this the baked version of sociology?

**Comment by oscryan** [comment]
Yes, and yes. Link above.

**Comment by nathanstitt** [comment]
We hide the sections so they're more compact in the reference view.  The screenshot is from QA view which is only partially styled.

Here's how it appears in reference view:

![image.png](https://images.zenhubusercontent.com/5b6c72f28ef805470660da46/06d48608-e9f0-42cf-bc06-662b55f40804)

We could show the chapter-sections but it'll take up more space. this is roughly what it'd look like:
![image.png](https://images.zenhubusercontent.com/5b6c72f28ef805470660da46/b5e57a85-b08b-451b-a27a-0447c7093d4c)

Note to myself - if we show them the styles for a.os-chapter-section would be roughly:
```
    margin: 0;
    margin-left: -140px;
    min-width: calc(100% + 280px);
    color: white;
    padding: 1rem;
    display: block;
    padding-left: 4rem;
```

**Comment by carlycurran** [comment]
@nathanstitt I think the first version works -- but where is the "Learning Objectives" header? I think that's the only thing missing as far as our usual styling.

**Comment by nathanstitt** [comment]
@carlycurran Yeah, we hide that also. Not sure why though, if we show it it'd look like:
![image](https://user-images.githubusercontent.com/79566/67584841-18dc2d00-f714-11e9-9b9e-ae5f69efcaab.png)

**Comment by nathanstitt** [comment]
also keep in mind that that isn't always "Learning Objectives", for instance Physics calls it "Chapter Outline"

![image](https://user-images.githubusercontent.com/79566/67586401-2810aa00-f717-11e9-8928-b82b78d0d198.png)
